### PR TITLE
feat: securely templated overrides 

### LINKED ISSUE #221: Feature Request: Saving docker-compose.override.yml files within settings
**Is your feature request related to a problem? Please describe.**
Only settings are saved, if docker-compose override files are created they are currently never externally saved.

**Describe the solution you'd like**
Move docker-compose override files to the settings folder like
`settings/overrides/servicename/docker-compose.override.yml`

**Describe alternatives you've considered**
Another folder to save them to.

**Additional context**

### PR DESCRIPTION
**Pull request by codefriar**
closes #221 

provides a way for services to be customized while still using encrypted template variables.

### CODE REVIEW

File: roles/vivumlab_service-edit/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+- name: Ensures override dir exists
+  file:
+    path: "settings/{{config_dir}}/override/{{ services[0] }}"
+    state: directory
+    owner: '1000'
```
Comment (denis-ev) [inline roles/vivumlab_service-edit/tasks/main.yml:6]: Is the owner etc needed here? It's running on localhost?

**Review by denis-ev — COMMENTED** [review]
Can you rebase this? There are so many dockercache changes/commits that shouldn't be in here.

**Comment by codefriar** [comment]
Without it, the file is unreadable/editable outside the docker container.

File: roles/vivumlab_service-edit/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    group: '1000'
+
+- name: Create docker-compose.override.yml
+  copy:
```
Comment (denis-ev) [inline roles/vivumlab_service-edit/tasks/main.yml:10]: If we template them, we need a template here instead of copy.
```suggestion
  template:
```

**Review by denis-ev — APPROVED** [review]

**Pull request merged**